accounts/api/views.py

```diff
@@ -49,7 +49,7 @@ def post(self, request):
         if Customer.objects.filter(user=user).exists():
             serializer = CustomerLoginSerializer(data=request.data)
 
-        elif Restaurant.objects.filter(user=user).exists():
+        elif Vendor.objects.filter(user=user).exists():
             serializer = RestaurantLoginSerializer(data=request.data)
         else:
             return Response({'error': 'User Not Register'}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
You don’t need to use “else” at the end of the line. Just return the Response. This is a Python convention.